test/resources/Charges.spec.js

```diff
@@ -120,9 +120,9 @@ describe('Charge Resource', function() {
     });
 
     it('Throws an error on incorrect arguments', function() {
```

Comment:
Nit: Maybe change the description since it doesn't throw anymore? Maybe something like `Incorrect arguments result in an error`.